### DISCUSSION TITLE
Branch capture_RPiHQ_li Stand 07.11.2021

### DISCUSSION THREAD

**Discussion by hoetzgit** [AndreasLMeg/allsky#8] [General]
Hallo Andreas,

ich teste auch mal Deine Anpassungen des Capture Teils und dem Keogram in meiner Installation. Betreibe einen Raspberry mit der HQ Kamera. Gefällt mir bisher sehr gut. Seite: https://allsky01.ott24.net

Grüße Henry

**Comment by AndreasLMeg** [comment]
Hallo Henry,

habe deine Nachricht erst jetzt gesehen. Danke für die Rückmeldung.
Deine Seite schaut aber auch sehr gut aus. 

Ist es Absicht, dass dur die Belichtungszeit und den gain so niedrig wählst. Die Bildqualität ist aber top!

Das mit den eingeblendeten Bildinfos muß ich mir auch mal ansehen, habe aber noch ein Hardwareprojekt (motorisierte Fokusierung) im Laufen.

Bin schon auf deine Erfahrungen gespannt. Ganz astrein läuft es ja noch nicht - aber ich bin zufrieden - besser als die ASI/ZWO Regelung.

Schöne Grüße nach B ayern
Andreas

**Comment by hoetzgit** [comment]
Hallo Andreas,

ich habe in der GUI folgende Werte eingestellt: Exposure: 50000ms, Gain: 6 und Auto-Gain: yes. In der config.sh erstmal nur Deine defaults "-mean-value 0.3 -autoexposure 1". Den Rest machen Deine Berechnungen oder? Habe das so verstanden, dass sich diese Werte dann im Rahmen der vorgegebenen Werte (Exposure/Gain) justieren.

Mir gefällt das Resultat jedenfalls sehr gut, zumal meine örtlichen Gegebenheiten sicher nicht optimal sind (Lichtverschmutzung etc).

Mit den Textinfos habe ich das vorläufig mit einem Bash-Skript gelöst. Rufe es im capture_RPiHQ.cpp direkt nach RPiHQcapture auf. saveImagexxx.sh habe ich dort auskommentiert. Mein Bash-Skript bekommt ein paar Parameter aus capture_RPiHQ.cpp mit, dann suche ich mir noch ein paar weitere Daten zusammmen, mache einen convert und rufe anschließend die jeweiligen saveImagexxx.sh auf. Keine Ahnung ob's so optimal ist, aber es funktioniert erstmal und ich muss nicht viele Anpassungen nach einem git pull machen.

Habe eh ein etwas seltsames Konstrukt daheim, der RPi hat das Image Verzeichnis auf einer Qnap, gemountet, die Erstellung des Videos und der Keogramme/Startrails lasse ich dann in einer Ubuntu VM auf einem meiner ESX Server laufen, dort sind auch die Verzeichnisse der Qnap gemountet. Deshalb sind mir neulich auch die Probleme unter Ubuntu beim make mit den CV Libs aufgefallen, obwohl ich auf der VM natürlich nicht den Capture Teil benötige. Die Webseite ist hinter einem NGINX Reverse Proxy selbst gehostet und läuft in einer weiteren VM.

Dein Motorfokus Projekt klingt spannend, habe lange gebraucht den Fokus manuell einigermaßen einzustellen. Schön wäre noch eine Meteorerkennung, die am Ende der Nacht dann ein Bild aus den erkannten Meteoren zu einem "zusammenbacken" würde ;-)
 
Dir weiterhin die Zeit und viel Spaß bei weiteren Entwicklungen
Viele Grüße
Henry
